### PR TITLE
:sparkles: Update k8s version in envtest

### PR DESCRIPTION
**Pull request by varshaprasad96**
Update k8s version in envtest for 0.7.0 release.

cc: @estroz (#1258)

### CODE REVIEW

File: pkg/internal/testing/integration/internal/integration_tests/etcd_integration_test.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ var _ = Describe("Etcd", func() {
 		// for on StdErr
 		Expect(etcd.Start()).To(MatchError(ContainSubstring("timeout")))
 
-		Expect(stdout.String()).To(ContainSubstring("member flags"))
-		Expect(stderr.String()).To(ContainSubstring("usage: etcd"))
+		Expect(stdout.String()).To(ContainSubstring("Member:"))
+		Expect(stderr.String()).To(ContainSubstring("Usage:"))
```
Comment (varshaprasad96) [inline pkg/internal/testing/integration/internal/integration_tests/etcd_integration_test.go:62]: The help text for etcd seems to have this format:
```
Expected
      <string>:
      Member:
        --name 'default'
       ...
```

and:

```
  Expected
      <string>: Usage:

        etcd [flags]
          Start an etcd server.
```

**Review by estroz — APPROVED** [review]
/lgtm

**Review by alvaroaleman — APPROVED** [review]
/approve

**Pull request merged**